### ISSUE TITLE
[Bug]: 2 Bugs on Mac M3 Pro Max

### ISSUE THREAD

**Issue by nosthas** [beyondcode/herd-community#838]
### Platform

macOS

### Operating system version

macOS Sonoma 14.5

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.7.1

### Bug description

The New site wizard is unable to create sqlite file so all new sites show an error page (SQLITE File not found). If the site is created from terminal everything is ok.
Also clicking on logs makes the app crash.

### Steps to reproduce

1) Click New Site from template
2) Click Open in browser

It seems the wizard do not show the database options before creating the site. 

### Relevant log output

```shell
Internal Server Error

Illuminate\Database\QueryException
Database file at path [/Users/xxxxx/Herd/ttddd/database/database.sqlite] does not exist. Ensure this is an absolute path to the database. (Connection: sqlite, SQL: select * from "sessions" where "id" = 96cZ1zsYgM0lrECqpIvffqctolvdin6MzdNFk948 limit 1)
```

**Comment by sschlein** [comment]
That's actually a problem with the Laravel installer that got changed a few days ago – it'll be fixed in the upcoming Herd version later this week. Logs have also been fixed, it looks like your current site does not have the logs folder where we expect it.

So, both are fixed in 1.8 🙂

**Comment by nosthas** [comment]
Thanks a lot
